packages/perseus/src/types.ts

```diff
@@ -15,6 +15,7 @@ import type {
     UserInputMap,
 } from "./validation.types";
 import type {WidgetPromptJSON} from "./widget-ai-utils/prompt-types";
+import type getCategorizerPublicOptions from "./widgets/categorizer/categorizer.util";
```

Comment:
Not sure if it's the right move to have these in their own file and, if that is the right move, wasn't sure about the filename being `categorizer.util.ts`. If anyone has thoughts on where the function and type should be, let me know :)

Reply:
It feels a little incorrect to call it "splitCategorizer" as we're not really splitting the whole widget in the same way we would score or validate it, so I changed it to "splitCategorizerWidgetOptions" and updated the rest accordingly. Also didn't want it to be "splitter" similar to how we do "scorer". 😅